sw/host/hsmtool/README.md

```diff
@@ -0,0 +1,5 @@
+# OpenTitan HSM Tool
+
+The OpenTitan HSM tool can be used to generate and manage keys used for OpenTitan signing ceremonies.
```

Comment:
```suggestion
The OpenTitan Hardware Security Module (HSM) tool can be used to generate and manage keys used for OpenTitan signing ceremonies.
```

Do we have a glossary of terms? If so maybe HSM should go in there; it's pretty general.

Reply:
Yeah, OpenTitan definitely needs a glossary, but I'll open an issue for it rather than make one in this PR

Reply:
Opened an issue for the glossary here: https://github.com/lowRISC/opentitan/issues/19437